### PR TITLE
New post regarding Twitter's Data download Feature

### PR DESCRIPTION
**Pull request by Xndr7**
References #90 
Hope it helps. If there are any further changes to be made, please tell me so that I can make them. I removed the section on porting of data because I couldn't find any site that accepted the data. I also used a free to use icon. Sorry for the separate pull request. I made an error in the commits and deleted my fork by mistake, I will close the earlier one #147 .

### CODE REVIEW

File: content/blog/twitter-data-download-explanation.en.md [hunk 1]
```diff
@@ -0,0 +1,906 @@
...
+    "type": "blog",
+    "slug": "twitter-data-download-explanation",
+    "date": "2019-10-17T16:58:19+02:00",
+    "description": "Twitter provides the option to download the user's personal data from their settings. This post explains the process to get the data as well as the meaning of the data provided.",
+    "featured_image": "/card-icons/Twitter_Black.svg",
+    "tags": [ "twitter", "gdpr", "data", "download", "export" ],
+    "authors": [ "xndr" ]
+}
+
+<img class="offset-image offset-image-left" src="/card-icons/Twitter_Black.svg" style="height: 200px;">
```
Comment (baltpeter) [inline content/blog/twitter-data-download-explanation.en.md]: Thanks for looking for another logo. However, I don't think this one adds any value as users don't know it.

I am currently working on a new 'framework' for featured images anyway, so please just remove the logo for now. I will add an image later.

File: content/blog/twitter-data-download-explanation.en.md [hunk 2]
```diff
@@ -0,0 +1,906 @@
...
+
+Twitter collects the users personal data to create and serve ads to them. For this purpose, it collects relevant user data and stores it to determine user interest and likes. This data can be a treasure trove of information on any frequent user. If the user opts in, the data is also shared with other partners as well. Twitter also stores a profile on any non logged-in user as well to serve ads based on their browsing habits. This is done on the device only and this data changes once the user logs in. The data stored is used to infer identity and serve ads as well as can be shared with other partners of Twitter.
+This data can be viewed by the user thanks to Article 15 GDPR which gives the right to data access. Know more about your rights in [this article]({{< ref "/blog/your-gdpr-rights" >}}). The method to access user data from such sites usually is to use the [generator]({{< ref "/generator" >}}).
```
Comment (baltpeter) [inline content/blog/twitter-data-download-explanation.en.md]: > The method to access user data from such sites usually is to use the [generator]({{< ref "/generator" >}}). 

That's flattering but not actually true. :) We offer the generator to users but most users certainly don't know about our generator. You can also just send the request yourself.

Comment (baltpeter) [reply content/blog/twitter-data-download-explanation.en.md]: Please also mention that the download won't include all data.

File: content/blog/twitter-data-download-explanation.en.md [hunk 3]
```diff
@@ -0,0 +1,906 @@
...
+  
+  
+
+
+{{< img src="/img/blog/twitter-data-explanation/non-login-twitter.png" caption="" >}}  
+
+
+<a id="twitter-logged-in-procedure"></a>
+## The Procedure to get the Data Download from Twitter (Logged in user)
+</a>
```
Comment (baltpeter) [inline content/blog/twitter-data-download-explanation.en.md]: The `</a>` needs to be removed.

File: content/blog/twitter-data-download-explanation.en.md [hunk 4]
```diff
@@ -0,0 +1,906 @@
...
+  
+  
+  
+  
+
+
+{{< img src="/img/blog/twitter-data-explanation/non-login-twitter.png" caption="" >}}  
+
+
+<a id="twitter-logged-in-procedure"></a>
```
Comment (zner0L) [inline content/blog/twitter-data-download-explanation.en.md]: You can do this via a heading anchor: https://gohugo.io/content-management/cross-references/#hugo-heading-anchors

**Comment by zner0L** [comment]
I will close this as there has been no feedback for quite a while. Feel free to reopen when you want to implement the requested changes.

**Pull request closed without merging**